modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_conjunction.scss

```diff
@@ -1,4 +1,28 @@
 .conjunction-button {
-	margin: 8px 0 8px 24px;
-	text-transform: uppercase;
+	text-transform: capitalize;
+	margin: {
+		bottom: $gap * 2;
```

Comment:
Hmm, I've never seen this syntax before. Could you point me to some docs on this?

Reply:
Actually I just found it. I talked to the clay-css team about it and I guess this syntax is not used in portal at all and is deprecated. It'd be best to replace it with something like...

```scss
margin-bottom: $gap * 2;
margin-top: $gap * 2;
```